identity-service/sequelize/migrations/20210218204013-create-recaptcha-scores.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('RecaptchaScores', {
```

Comment:
to make this more extensible, what if we made this table called BotScores and renamed the fields scores, context, and hostname to recaptchaScores recaptchaContext and recaptchaHostname or something like that? or maybe even added a type column

Reply:
i figure if these column names are in the table RecaptchaScores (or BotScores), i think the context is clear on the purpose of these columns/not necessary to have a column name change? i don't feel strongly about this so im ok with either option. and what would a type column be for?

Reply:
discussed offline -- making columns more specific allows for easier potential migrations in future if necessary. will change

Reply:
discussed offline -- keep all records VS updating a row of a user for a more recent score